app/src/pages/inside/projectSettingsPageContainer/content/notifications/modals/addEditNotificationModal/addEditNotificationModal.jsx

```diff
@@ -356,7 +356,7 @@ const AddEditNotificationModal = ({
             </FieldElement>
             <FieldElement
               name={INFORM_OWNER_FIELD_KEY}
-              type="text"
+              type="checkbox"
```

Comment:
The `Checkbox` already has type `checkbox`.
Can we just remove this line?
And looks like `type="text"` also redundant for all other fields within this component.